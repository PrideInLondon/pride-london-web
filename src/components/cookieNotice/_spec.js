import React from 'react'
import { shallow } from 'enzyme'
import CookieNotice from '.'

describe('<CookieNotice />', () => {
  it('renders and matches snapshot', () => {
    const wrapper = shallow(<CookieNotice />)
    expect(wrapper).toMatchSnapshot()
  })
})
