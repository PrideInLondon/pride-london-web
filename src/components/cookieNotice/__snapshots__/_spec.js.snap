// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CookieNotice /> renders and matches snapshot 1`] = `
<Fragment>
  <cookieNotice__CookieWrapper
    hide={true}
  >
    <grid__Container
      mx="auto"
    >
      <Row
        flexWrap={null}
        ml={null}
        mr={null}
        mx={null}
      >
        <Column
          pb={null}
          pl={null}
          pr={null}
          pt={null}
          px={null}
          py={null}
          width={
            Array [
              1,
              1,
              0.6,
            ]
          }
        >
          <cookieNotice__CookieText>
            We use cookies to make sure that you have the best experience on our website and to learn how we can make it better for you.
             
            <cookieNotice__CookieLink
              to="/privacy"
            >
              Learn more
            </cookieNotice__CookieLink>
          </cookieNotice__CookieText>
        </Column>
        <cookieNotice__CookieActions
          width={
            Array [
              1,
              1,
              0.4,
            ]
          }
        >
          <cookieNotice__CookieAgree
            onClick={[Function]}
            role="button"
          >
            Got it
          </cookieNotice__CookieAgree>
          <cookieNotice__CookieDismiss
            aria-label="I do not agree and close"
            onClick={[Function]}
          >
            <CloseIcon />
          </cookieNotice__CookieDismiss>
        </cookieNotice__CookieActions>
      </Row>
    </grid__Container>
  </cookieNotice__CookieWrapper>
</Fragment>
`;
